clusterloader2/cmd/clusterloader.go

```diff
@@ -159,6 +161,20 @@ func main() {
 		glog.Fatalf("Parsing flags error: %v", errList.String())
 	}
 
+	// TODO: Replce glog with klog.
+	// Init klog.
+	klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
+	klog.InitFlags(klogFlags)
+
+	// Sync the glog and klog flags.
```

Comment:
IIUC, we can get rid of this once we replace glog with klog in cluster loader? If so, could you document that?

Reply:
Comment added.